src/lib/components/forms/file-input.svelte

```diff
@@ -0,0 +1,35 @@
+<script>
+  export let name = '';
+
+  let files = [];
+  function append_file(event) {
+    files = Array.from(event.target.files);
+  }
+  function pop_file(fileRemove) {
```

Comment:
```suggestion
  function appendFile(event) {
    files = Array.from(event.target.files);
  }
  function popFile(fileRemove) {
```

For consistency with other files